wazo_load_api/etc/wazo-load-api/private.key

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
Is it a concern that this file is public?

Reply:
Not a problem due to the fact we generate it on demand for testing purpose, this one particularly won't be used anymore.